disnake/channel.py

```diff
@@ -2488,7 +2497,29 @@ class ForumChannel(disnake.abc.GuildChannel, Hashable):
         The default auto archive duration in minutes for threads created in this channel.
     slowmode_delay: :class:`int`
         The number of seconds a member must wait between creating threads
-        in this channel. A value of `0` denotes that it is disabled.
+        in this channel.
+
+        A value of ``0`` denotes that it is disabled.
+        Bots and users with :attr:`~Permissions.manage_channels` or
+        :attr:`~Permissions.manage_messages` bypass slowmode.
```

Comment:
Actually, bots are always exempt.

Reply:
The wording was somewhat ambiguous and could've meant two different things, changed it to be similar to the api docs in https://github.com/DisnakeDev/disnake/pull/726/commits/add07c228eb18c65a89a5999f1fe85fb4e8d30e0.